plutus-benchmark/plutus-benchmark.cabal

```diff
@@ -550,13 +550,25 @@ test-suite plutus-benchmark-marlowe-tests
 
 -- TODO: Add benchmarks for the executable semantics when we have a UPLC version
 
+library agda-internal
+  import:          lang, ghc-version-support
+  ghc-options:     -Wno-unused-packages
```

Comment:
I wonder why this is needed? Would be nice to have an answer in a comment.

Reply:
I'll take that out again: initially it wasn't there and when I started getting the failures I put it in to see if it helped since it was in some of the other things in the file.

Reply:
@yura: `-Wno-unused-packages` was scattered about in various places in the cabal file but now I've worked out exactly where it's needed and introduced a sort of alias that makes it a bit clearer what's going on (although I'm still not totally sure what _is_going on).  I'd forgotten why we needed this, so thanks for making me think about it.